e3sm_to_cmip/__main__.py

```diff
@@ -651,25 +651,30 @@ def _run_info_mode(self):  # noqa: C901
         messages = []
 
         # if the user just asked for the handler info
-        if self.freq == "mon" and not self.input_path and not self.tables_path:
+        # TONY SAYS: Why does this mode even exist?  And why does it depend upon "freq = mon"?
+        if self.freq == "mon" and not self.input_path and not self.tables_path: # info mode 1
             for handler in self.handlers:
                 hand_msg = get_handler_info_msg(handler)
                 messages.append(hand_msg)
 
         # if the user asked if the variable is included in the table
         # but didnt ask about the files in the inpath
-        elif self.freq and self.tables_path and not self.input_path:
+        elif self.freq and self.tables_path and not self.input_path:    # info mode 2
             for handler in self.handlers:
                 table_info = _get_table_info(self.tables_path, handler["table"])
                 if handler["name"] not in table_info["variable_entry"]:
                     msg = f"Variable {handler['name']} is not included in the table {handler['table']}"  # type: ignore
                     print_message(msg, status="error")
                     continue
                 else:
+                    if self.freq == "mon" and handler['table'] == "CMIP6_day.json":
+                        continue
+                    if ( self.freq == "day" or self.freq == "3hr" ) and handler['table'] == "CMIP6_Amon.json":
+                        continue
```

Comment:
From what I understand, this skips trying to print handlers that have CMIP tables that don't match the selected `freq`? If so, what is the purpose of skipping (is it intentional and/or should we print some message?).

Reply:
We skip it because of the nature of the search method (look at them all, see which is a match).  If you allow a user to specify `freq`, you should not report the non-matching values.  Otherwise, why allow the user to specify `freq`?

If you wrote "grep", and look for lines in a file that contain "ERROR", you don't print a message regarding the lines that don't contain "ERROR".